frontend/src/views/CanteensPage/ContactForm.vue

```diff
@@ -63,6 +67,11 @@ export default {
             status: "success",
             message: `Votre message a bien été envoyé à ${this.canteen.name}. Merci de vôtre intérêt.`,
           })
+
+          if (this.$matomo) {
+            this.$matomo.trackEvent("message", "send", "cantine-email-contact")
```

Comment:
```suggestion
            this.$matomo.trackEvent("message", "send", "canteen-email-contact")
```
Est-ce que tu as un idée de comment on veut gérer les premier deux arguments ? Par exemple, j'ai utilisé `form` et `submit` pour le générateur d'affichage qui peuvent être réutilisés ici.